### ISSUE TITLE
Paasta status for stopped instance says "Running"

### ISSUE THREAD

**Issue by asottile** [Yelp/paasta#587]
```
$ paasta status -s error_pages -c norcal-stagef
Pipeline: https://jenkins.yelpcorp.com/view/services-error_pages

cluster: norcal-stagef
instance: main
Git sha:    4b8da1f5 (desired)
State:      Running - Desired state: Stopped
Marathon:   Healthy - up with (0/0) instances. Status: Running.
Mesos:      Healthy - (0/0) tasks in the TASK_RUNNING state.
Smartstack:
    norcal-stagef - Healthy - in haproxy with (0/0) total backends UP in this namespace.
    
```

**Comment by solarkennedy** [comment]
This mirrors internal ticket PAASTA-123

**Comment by keshavdv** [comment]
Fixed by #590
